### PR TITLE
Add new time attribute to the 'insightsoperator' CRD

### PR DESCRIPTION
**Pull request by tremes**
This adds a new time attribute (providing the last time when the Insights report was downloaded) to the `insightsoperator` status CRD.

### CODE REVIEW

File: operator/v1/types_insights.go
Comment (JoelSpeed) [inline operator/v1/types_insights.go:65]: Nit sentences should finish with a full stop

What does it mean for this to be empty?

Comment (tremes) [reply operator/v1/types_insights.go]: Updated. THanks

Comment (tremes) [reply operator/v1/types_insights.go]: Oh let me regenerate it...

Comment (JoelSpeed) [reply operator/v1/types_insights.go]: When is this field likely to be empty and what does it mean for it to be empty?

Comment (tremes) [reply operator/v1/types_insights.go]: The field will be empty until there's some Insights data upload and consecutive Insights analysis report download. For example it can be empty in clusters when the Insights operator is disabled (disconnected).

Comment (JoelSpeed) [reply operator/v1/types_insights.go]: Can you make sure to explain that within the godoc please

**Comment by JoelSpeed** [comment]
Can we get an SME from the insights side to review this as well please

**Comment by JoelSpeed** [comment]
/approve cancel

I should hold the approve until after the SME review, but, API wise, this is good to go I think. Please get an LGTM and drop me a note

**Review by rluders — APPROVED** [review]
/lgtm

**Pull request merged**
